sdk/storage/Azure.Storage.Files.DataLake/src/Models/PathPermissions.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Text;
+
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// Represents POSIX-style permissions on a given resource. Each resource specifies permissions for the owner, the owning
+    /// group, and everyone else. Permissions for users or groups not included here can be set using an Access Control List.
+    /// Manipulating resource permissions is only supported when ADLS interop is enabled.
+    /// </summary>
+    public class PathPermissions
+    {
+        /// <summary>
+        /// The <see cref="RolePermissions"/> for the owner of the resource.
+        /// </summary>
+        public RolePermissions Owner { get; set; }
+
+        /// <summary>
+        /// The <see cref="RolePermissions"/> for the owning group of the resource.
+        /// </summary>
+        public RolePermissions Group { get; set; }
+
+        /// <summary>
+        /// The <see cref="RolePermissions"/> for the other users.
+        /// </summary>
+        public RolePermissions Other { get; set; }
+
+        /// <summary>
+        /// If the sticky bit has been set. The sticky bit may be set on directories, the files in that
+        /// directory may only be renamed or deleted by the file's owner, the directory's owner, or the root user.
+        /// </summary>
+        public bool StickyBit { get; set; }
+
+        /// <summary>
+        /// Whether or not there is more permissions information in the ACLs. The permissions string only returns
+        /// information on the owner, owning group, and other, but the ACLs may contain more permissions for specific users
+        /// or groups.
+        /// </summary>
+        public bool ExtendedInfoInAcl { get; set; }
```

Comment:
Should this just be `ExtendedAcls`?

Reply:
I think the current property name is more descriptive.

Reply:
The current name reads a little weird and doesn't capture what various online sources say that `+` implies (i.e.,  https://en.wikipedia.org/wiki/File_system_permissions#Symbolic_notation says "+ (plus) suffix indicates an access control list that can control additional permissions.")

Reply:
fixed.